src/EditorApp.tsx

```diff
@@ -94,8 +89,6 @@ const EditorApp: React.FC<EditorAppProps> = ({
       manuscriptID,
       files,
       permittedActions,
-      userID: userID || '',
```

Comment:
Maybe I'm missing something but it looks like userID is not getting to the store anymore. If so, can you please remove it from the state type and check if the app builds ok without it. (see src/store/Store.ts around line :39)

Reply:
Please remove it from the store then. Thanks